### ISSUE TITLE
App shows blank screen when trying to open task module on Outlook desktop beta app

### ISSUE THREAD

**Issue by ktcodebite** [MicrosoftDocs/msteams-docs#8234]
Reporting on behalf of partner:

Outlook web Jira Cloud app is working without any issues and task modules are displayed
But Outlook desktop (Windows) we can see just blank screen while trying to open any task module.

- the loading hanging  while we trying to obtain SSO token by calling teams-js  client library 
`await microsoftTeams.authentication.getAuthToken({ silent: true })`

- Teams context for that case is: {“host”: {“name”:”OutlookWin32”, “clientType”:”desktop”}, “page”: {“frameContext”:”task”}}
- We do not have issue for Teams context : {“host”: {“name”:”OutlookWin32”, “clientType”:”desktop”}, “page”: {“frameContext”:”content”}} 
- I was not able to catch any error. Execution just stopped while we trying to get token
- When I hardcode the token value, the task module was displayed correctly
 
![jiracloud](https://user-images.githubusercontent.com/16550774/222538477-80c15539-1c48-431a-a36f-ee4fcfac4a9c.png)

**Comment by Nivedipa-MSFT** [comment]
@ktcodebite - Thanks for reporting your issue. Could you please share the TeamsJS library version which you are using to test at our end?

**Comment by vbatr** [comment]
@Nivedipa-MSFT  that is reproducible on  "@microsoft/teams-js": "^2.6.0"

**Comment by vbatr** [comment]
@Nivedipa-MSFT  on the latest  v 2.9.0 [dialog.open()](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fjavascript%2Fapi%2F%40microsoft%2Fteams-js%2Fdialog.url%3Fview%3Dmsteams-client-js-latest%23%40microsoft-teams-js-dialog-url-open&data=05%7C01%7CKiran.Thomas%40microsoft.com%7Ccbae8e6494934b8077fc08db1b43a2b9%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638133752360798187%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000%7C%7C%7C&sdata=mGCIXIz%2BtH93Dppapd6yijWFSlAZ5XJCpoPv%2BPe9rmQ%3D&reserved=0) is not working at all, so we cannot even open the task module. The issue is reproducible on Teams web, Outlook web, Outlook desktop beta

**Comment by Nivedipa-MSFT** [comment]
@ktcodebite - We have tested dialog.open() on Teams web, Outlook web and Outlook desktop beta version 2302 using TeamJs version 2.9.0.
We are able to repro your issue and raised the bug for the same. We will let you know once we get any updates on it.
[dialog.open 2.9.0.webm](https://user-images.githubusercontent.com/92150070/222716531-c85f1f6c-ce82-4934-b131-dac730d15b8d.webm)

**Comment by vbatr** [comment]
@Nivedipa-MSFT , @Wajeed-msft  thanks for checking the issue. 
Have you also tested scenario with calling  microsoftTeams.authentication.getAuthToken() from Outlook desktop task module (lib v 2.6.0) - the original issue reported by @ktcodebite ?
We are pretty sure it will be still reproducible once dialog.open() issue is fixed on newer versions of the library.

**Comment by vbatr** [comment]
@Nivedipa-MSFT  thank you for checking that. And just to confirm: Have you run this code inside opened task module? As it is working as expected for “frameContext”:”content” but not for  “frameContext”:”task”

**Comment by Nivedipa-MSFT** [comment]
@ktcodebite, @vbatr - We are able to repro your issue. We have called getAuthToken() on TaskModule. Its working on Teams web client and not in Outlook desktop version 2304.

Teams web client:
[Teams-Web.webm](https://user-images.githubusercontent.com/92150070/226583057-f9e2e726-d8d1-4465-86c9-ca8252f36835.webm)

Outlook desktop version 2304:
[Outlook-desktop.webm](https://user-images.githubusercontent.com/92150070/226583112-cd66f7a3-fc04-4ad0-bbaf-1676d3d8f679.webm)

We have raised a bug for this. We will let you know once we get any updates on it.
